cl/search/management/commands/cl_index_parent_and_child_docs.py

```diff
@@ -199,7 +199,11 @@ def get_documents_to_update_or_remove(
     child_documents_to_update = []
     documents_to_delete = []
     event_ids = list(events_to_update.values_list("id", flat=True))
-    for event_ids_chunk in chunks(event_ids, chunk_size):
+    event_ids_count = len(event_ids)
+    processed_count = 0
+    for i, event_ids_chunk in enumerate(
+        chunks(event_ids, chunk_size), start=1
```

Comment:
I missed it before, but we should put a warning in our code to stop using `chunks`, since it's built into Python these days: https://docs.python.org/3/library/itertools.html#itertools.batched